defs/tests/scenarios/storage/ceph/ceph-mon/PG_overdose.yaml

```diff
@@ -0,0 +1,33 @@
+data-root:
```

Comment:
can you make the filename all lowercase since thats the convention i.e. ceph-mon/pg_overdose.yaml

Reply:
done